client/my-sites/upgrades/checkout-thank-you/features-header.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import {
+	isDomainMapping,
+	isDomainRegistration,
+	isGoogleApps
+} from 'lib/products-values';
+
+const FeaturesHeader = ( { isDataLoaded, isGenericReceipt, purchases } ) => {
+	const classes = classNames( 'checkout-thank-you__features-header', {
+		'is-placeholder': ! isDataLoaded
+	} );
+
+	if ( ! isDataLoaded ) {
+		return <div className={ classes } />;
+	}
+
+	if ( isGenericReceipt ) {
```

Comment:
I'm not sure this is a valid reason. The `null` thing is clearly a bug in React and will be fixed in future versions